telegram/_bot.py

```diff
@@ -4973,12 +4973,20 @@ async def send_invoice(
                 :tg-const:`telegram.Invoice.MAX_PAYLOAD_LENGTH` bytes. This will not be
                 displayed to the user, use for your internal processes.
             provider_token (:obj:`str`): Payments provider token, obtained via
-                `@BotFather <https://t.me/BotFather>`_.
+                `@BotFather <https://t.me/BotFather>`_. Pass an empty string for payments in
+                `Telegram Stars <https://t.me/BotNews/90>`_.
+
+                .. deprecated:: NEXT.VERSION
+                    As of Bot API 7.4, this parameter is now optional and future versions of the
+                    library will make it optional as well.
+
```

Comment:
just wondering: TG marked this as optional (i.e. it's allowed to completely omit), but says it wants an empty string for TG stars - so it's required after all or does TG convert None/Null to ""?
Did not find the question on the bot api repo yet https://github.com/tdlib/telegram-bot-api/issues?q=is%3Aissue+is%3Aopen+provider_token

Reply:
that's what I was thinking too. I passed `None` for that argument and it worked as well.

Reply:
https://t.me/c/1494805131/41640

> Empty string and omitting is equivalent unless otherwise documented